internal/vm/checker/checker.go

```diff
@@ -17,8 +17,6 @@ import (
 	"github.com/google/mtail/internal/vm/types"
 )
 
-const kMaxRegexpLen = 1024
```

Comment:
Keep this but rename to 'kDefaultMaxRegexpLength' and add 'kDefaultMaxRecursionDepth', see below for why